src/main/java/uk/gov/pay/directdebit/payments/dao/mapper/TransactionMapper.java

```diff
@@ -13,12 +13,14 @@
     private static final String AMOUNT_COLUMN = "amount";
     private static final String TYPE_COLUMN = "type";
     private static final String STATE_COLUMN = "state";
+    private static final String PAYMENT_REQUEST_EXTERNAL_ID = "external_id";
```

Comment:
If using all with suffix COLUMN then this one needs to be updated (for next PR is fine)